crates/exex/src/serde.rs

```diff
@@ -105,7 +146,7 @@ mod tests {
 
     fn setup_kakarot_serde() -> KakarotSerde {
         // Load the valid program content from a JSON file
-        let program_content = include_bytes!("../../../cairo/programs/os.json");
+        let program_content = include_bytes!("../testdata/os.json");
```

Comment:
@ClementWalter I know that here you preferred that I directly use the OS program located in the Cairo part. But I isolated it in the Rust part for the following reasons:
- I am currently testing the serialization functions in a unitary way (without complete serialization of a realistic program).
- So I mock almost everything and I base myself only on the JSON (not on a complete Cairo logic, I will make at the end a complete test which will be "Cairo readable" to deserialize).
- As I base myself on these things so low level, as soon as we modify a single line of the Cairo OS and therefore its JSON, all our Rust unit tests will explode and it will take a crazy amount of time to fix them. 
- So basically this JSON file = we must never touch it